tests/channels.rs

```diff
@@ -360,3 +378,181 @@ fn can_set_all_on_off() {
     pwm.set_all_on_off(&on, &off).unwrap();
     destroy(pwm);
 }
+
+#[test]
+
+fn can_set_all_channels() {
+    let trans = [
+        I2cTrans::write(DEV_ADDR, vec![Register::MODE1, MODE1_AI]),
+        I2cTrans::write(
+            DEV_ADDR,
+            vec![
+                Register::C0_ON_L,
+                1,
+                1,
+                3,
+                3,
+                2,
+                1,
+                4,
+                3,
+                3,
+                1,
+                5,
+                3,
+                4,
+                1,
+                6,
+                3,
+                5,
+                1,
+                7,
+                3,
+                6,
+                1,
+                8,
+                3,
+                7,
+                1,
+                9,
+                3,
+                8,
+                1,
+                0,
+                4,
+                9,
+                1,
+                1,
+                4,
+                0,
+                2,
+                2,
+                4,
+                1,
+                2,
+                3,
+                4,
+                2,
+                2,
+                4,
+                4,
+                3,
+                2,
+                5,
+                4,
+                4,
+                2,
+                6,
+                4 + 16,
```

Comment:
Could you use a local constant for the full on/off like above here as well for the value `16`?